pkg/compose/exec.go

```diff
@@ -118,11 +118,13 @@ func (s *composeService) interactiveExec(ctx context.Context, opts api.RunOption
 			_, err := stdcopy.StdCopy(opts.Stdout, opts.Stderr, stdout)
 			outputDone <- err
 		}
+		defer stdout.Close() //nolint:errcheck
```

Comment:
Any reason this is in a `defer` as it’s the last statement in the function?

Reply:
nope, I just did the same as `stdin` without actually checking :P